src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleCloudStorageDAO.scala

```diff
@@ -177,7 +177,11 @@ class HttpGoogleCloudStorageDAO(
   override def getWorkspaces(userId: String): Seq[(WorkspaceName, WorkspaceAccessLevel)] = {
     val credential = getGroupServiceAccountCredential
     val directory = new Directory.Builder(httpTransport, jsonFactory, credential).setApplicationName(appName).build()
-    val workspaceTuples = directory.groups().list().setUserKey(userId).execute().getGroups.map(g => deconstructGroupId(g.getEmail))
+    val workspaceTuples = try {
+      directory.groups().list().setUserKey(userId).execute().getGroups.map(g => deconstructGroupId(g.getEmail))
+    } catch {
+      case e: NullPointerException => Seq.empty
+    }
```

Comment:
I feel like we shouldn't have a blanket ```catch``` over the whole statement. For example if ```directory``` is null (for some reason) then we'd want to bubble that up, instead of interpreting it as the user not being a member of any workspace.

Maybe do the ```execute``` statement, then check if ```getGroups``` is null?